packages/account/src/Configs/user-profile-validation-config.ts

```diff
@@ -29,8 +29,15 @@ const makeTinOptional = (
     employment_status: string,
     { is_mf, is_real, tin_skipped, is_tin_auto_set }: TINDepdendents
 ) => {
-    const should_not_bypass_tin = !tin_config?.tin_employment_status_bypass?.includes(employment_status);
-    return (tin_skipped && !is_tin_auto_set) || (!is_mf && !is_real) || !(is_real && should_not_bypass_tin);
+    const should_bypass_tin = tin_config?.tin_employment_status_bypass?.includes(employment_status);
+    const check_if_tin_skipped = tin_skipped && !is_tin_auto_set;
```

Comment:
what does `is_tin_auto_set` mean?
can we add a description as a comment

Reply:
tin_auto_set tells if tin was set to Aprooved000 earlier. Will add a comment